src/components/security_manager/src/security_manager_impl.cc

```diff
@@ -585,7 +585,7 @@ void SecurityManagerImpl::SendQuery(const SecurityQuery& query,
                                          protocol_handler::kControl));
     DCHECK(protocol_handler_);
     // Add RawMessage to ProtocolHandler message query
-    protocol_handler_->SendMessageToMobileApp(rawMessagePtr, false);
+    protocol_handler_->SendMessageToMobileApp(rawMessagePtr, false, false);
```

Comment:
@BSolonenko i think to maintain logic of previous implementation, we need to encrypt message depending on whether we have secure RPC service or not, since we are not sending here an RPC per se